recipes/metpy/meta.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+  name: metpy
+  version: !!str 0.3.0
```

Comment:
>  I don't know if this is necessary. @ocefpaf, do you typically just "0.3.0" it? Or even, just 0.3.0 directly.

Right, no need for that.

@dopplershift probably used `conda skeleton` as a template.  (Thee skeleton still creates the yaml file using `!! str` for version.)

> Either way, one simplification for maintenance of this recipe would be to set a variable up top

I don't like the jinja variable when the variable will be used only once like in the example.  Although, in the MetPy recipe, that variable will be used in 3 different places. (`url`, `fn`, and `version`.)